### ISSUE TITLE
Build unexpectedly succeeds inside of eic-shell

### ISSUE THREAD

**Issue by faustus123** [eic/EICrecon#72]
### Environment: (where does this bug occur, have you tried other environments)
- Which branch (often `main` for latest released): main
- Which version (or `HEAD` for the most recent on git): HEAD
- Any specific OS or system where the issue occurs? RHEL7
- Any special versions of ROOT or Geant4?

### Steps to reproduce: (give a step by step account of how to trigger the bug)
Trying to build EICrecon on REHL7 outside of eic-shell container is giving different results from inside container. Steps to build packages currently listed in top-level README.md for EICrecon. 

### Expected Result: (what do you expect when you execute the steps above)
Building in eic-shell seems to work just fine with no errors. I expect the same when building locally using similar package versions. 

Looking at the CMakeLists.txt however, I cannot see where ROOT is introduced to the _rootfile_ project. (It is not in the top-level CMakeLists.txt, nor in the CMakeLists.txt in the services/rootfile directory, nor in the cmake/jana_plugin.cmake file that it uses). Thus, I guess the error is the "expected" behavior and the success of the container build is unexpected(?)

### Actual Result: (what do you get when you execute the steps above)
I get the following error:
~~~
In file included from /home/davidl/work/2022.09.02.EICrecon/EICrecon/src/services/rootfile/rootfile.cc:6:
/home/davidl/work/2022.09.02.EICrecon/EICrecon/src/services/rootfile/RootFile_service.h:15:10: fatal error: TFile.h: No such file or directory
   15 | #include <TFile.h>
~~~

Here are some of the details:
1. I'm using root 6.26.04 which is newer than the container version of 6.22.06, but I don't think that is the issue
2. I'm using cmake 3.24.1 which is newer than the container version of 3.23.1, but I don't think that is the issue
3. cmake does not complain about being unable to find root despite several _find\_package(ROOT REQUIRED)_ lines

If I add these lines to _src/services/rootfile/CMakeLists.txt_, then it compiles just fine.
~~~
find_package(ROOT REQUIRED)
plugin_include_directories(${PLUGIN_NAME} SYSTEM PUBLIC ${ROOT_INCLUDE_DIRS})
~~~

So basically, why does it work in the container without those lines?

**Comment by faustus123** [comment]
Note: I just submitted a draft PR #73 that gets around the problem, but I'm not convinced this is the right solution

**Comment by faustus123** [comment]
Scratch that. Commit did not include the change that fixes it. Working on it ...

**Comment by faustus123** [comment]
eic-shell installs to /usr/local where root isfound

**Comment by cissieAB** [comment]
> eic-shell installs to /usr/local where root isfound
jug_xl> xx@localhost:~/eic/dev/EICrecon/lib/EICrecon/plugins$ /usr/bin/which root
/usr/local/bin/root
jug_xl> xx@localhost:~/eic/dev/EICrecon/lib/EICrecon/plugins$ ls -la /usr/local/bin/root
lrwxrwxrwx. 1 root root 99 Sep  1 02:16 /usr/local/bin/root -> /opt/software/linux-debian-x86_64/gcc-12.2.0/root-6.26.06-j2kqo6cciiraecjzk3htkx5c7ooklfh4/bin/root

**Comment by faustus123** [comment]
This was resolved some time ago. Not sure the PR#, but closing this now to clean things up a bit.
